Web pages

MaxVSCJP opened the pull request.

MaxVSCJP committed:
Capstone Project from imported from VS code

MaxVSCJP committed:
Small visual change to code

There were empty lines of code at the top of the main web page html file. Just got rid of that

DawitSishu reviewed (changes requested):
### Changes Requested:

Please ensure that the `div` with the class "Headline" has a height of `100vh`. Additionally, incorporate a media query to ensure responsiveness.

```css
.Headline {
  height: 100vh;
}

@media screen and (max-width: 768px) {
  /* Adjust styles here */
}

MaxVSCJP committed:
Last touch ups to
-make the main headline part take up the entire height of the page and,
-make the website responsive by adding a media query.

MaxVSCJP committed:
Update main style.css

MaxVSCJP commented:
> ### Changes Requested:
> Please ensure that the `div` with the class "Headline" has a height of `100vh`. Additionally, incorporate a media query to ensure responsiveness.
> 
> ```css
> .Headline {
>   height: 100vh;
> }
> 
> @media screen and (max-width: 768px) {
>   /* Adjust styles here */
> }
> ```

Thank you for the comment, changes have been made
